editor/src/components/canvas/canvas-strategies/drag-to-insert-strategy.tsx

```diff
@@ -196,6 +197,7 @@ function runTargetStrategiesForFreshlyInsertedElement(
   interactionState: InteractionSession,
   commandLifecycle: 'mid-interaction' | 'end-interaction',
   insertionSubjects: Array<{ command: InsertElementInsertionSubject; frame: CanvasRectangle }>,
+  strategyLifeCycle: 'mid-interaction' | 'end-interaction',
```

Comment:
can we factor this out into a type? (to avoid duplication)

Reply:
Good point, fixed in 31f76c49593651c9919a44ad73edddbbe00d2720